src/deepstream.io.js

```diff
@@ -255,18 +257,11 @@ Deepstream.prototype._showStartLogo = function() {
 		return;
 	}
 	/* istanbul ignore next */
-	var logo =
-	' _____________________________________________________________________________\n'+
-	'                                                                              \n'+
-	'         /                                                             ,      \n'+
-	' ----__-/----__----__------__---__--_/_---)__----__----__---_--_-----------__-\n'+
-	'   /   /   /___) /___)   /   ) (_ ` /    /   ) /___) /   ) / /  )    /   /   )\n'+
-	' _(___/___(___ _(___ ___/___/_(__)_(_ __/_____(___ _(___(_/_/__/__o_/___(___/_\n'+
-	'                       /                                                      \n'+
-	'                      /                                                       \n'+
-	'=============================== STARTING... ==================================\n';
+	var logo = fs.readFileSync( './ascii-logo.txt', 'utf8' );
+
 	/* istanbul ignore next */
-	process.stdout.write( this._options.colors ? logo.yellow : logo );
+	process.stdout.write( logo + EOL );
+	process.stdout.write( ' =========================   ' + ( this._options.colors ? 'starting'.cyan : 'starting' ) + '   ==========================' + EOL );
 };
```

Comment:
Maybe logo should be once log starts. Bit wierd when using only a file logger if deepstream only shows the logo in console.

Only time that isn't the case is any fails before launch